mythx_cli/fuzz/rpc.py

```diff
@@ -0,0 +1,120 @@
+import json
+import logging
+import os
+from pathlib import Path
+
+import click
+import requests
+from requests import RequestException
+
+from .exceptions import RPCCallError
+
+LOGGER = logging.getLogger("mythx-cli")
+
+headers = {"Content-Type": "application/json"}
+time_limit_seconds = 3000
+NUM_BLOCKS_UPPER_LIMIT = 9999
+
+
+class RPCClient:
+    def __init__(self, rpc_url: str, number_of_cores: int):
+        self.rpc_url = rpc_url
+        self.number_of_cores = number_of_cores
+
+    def call(self, method: str, params: str):
+        """Make an rpc call to the RPC endpoint
+
+        :return: Result property of the RPC response
+        """
+        try:
+            payload = (
+                '{"jsonrpc":"2.0","method":"'
+                + method
+                + '","params":'
+                + params
+                + ',"id":1}'
+            )
+            response = (
+                requests.request("POST", self.rpc_url, headers=headers, data=payload)
+            ).json()
+            return response["result"]
+        except RequestException as e:
+            raise RPCCallError(
+                f"HTTP error calling RPC method {method} with parameters: {params}"
+                f"\nAre you sure the RPC is running at {self.rpc_url}?"
+            )
+
+    def contract_exists(self, contract_address):
+        return self.call("eth_getCode", '["' + contract_address + '","latest"]')
+
+    def get_block(self, latest: bool = False, block_number: int = -1):
+        block_value = "latest" if latest else str(block_number)
+        if not latest:
+            block_value = hex(block_number)
+
+        block = self.call("eth_getBlockByNumber", '["' + block_value + '", true]')
+        if block is None:
+            return None
+        else:
+            return block
```

Comment:
If `block` is `None` or anything else, then we can simplify just to `return block`.

Reply:
Yeah, this was a brain fart :D